fix gh-1216 rewrite replicaset.callXX as replicaset_object:callXX (#1340)
tarantool/doc: doc/reference/reference_rock/vshard/vshard_ref.rst

        Call a function on a nearest available master (distances are defined using
        ``replica.zone`` and ``cfg.weights`` matrix) with a specified
        arguments.

        .. NOTE::

            The ``replicaset_object:callrw`` method is similar to ``replicaset_object:call``.

        :param function_name: function to execute
        :param argument_list: array of the function's arguments
        :param options:

            * ``timeout`` – a request timeout, in seconds. In case the ``router`` cannot identify a
              shard with the bucket id, the operation will be repeated until the
              timeout is reached.

            * other :ref:`net.box options <net_box-options>`, such as ``is_async``,
              ``buffer``, ``on_push`` are also supported.

        :return: 

            * result of ``function_name`` on success
            * nill, err otherwise

    .. _router_api-replicaset_callro:

    .. method:: callro(function_name, {argument_list}, {options})

        Call a function on the nearest available replica (distances are defined
        using ``replica.zone`` and ``cfg.weights`` matrix) with specified

>         arguments. It is recommended to call only read-only functions using

Reviewer: ```suggestion
        arguments. It is recommended to use
```

>         ``replicaset_object:callro()``, as the function can be executed not only

Reviewer: ```suggestion
        ``replicaset_object:callro()`` for calling only read-only functions, as the called functions can be executed not only
```

        on a master, but also on replicas.

        :param function_name: function to execute
        :param argument_list: array of the function's arguments
        :param options:

            * ``timeout`` – a request timeout, in seconds. In case the ``router`` cannot identify a
              shard with the bucket id, the operation will be repeated until the
              timeout is reached.

            * other :ref:`net.box options <net_box-options>`, such as ``is_async``,
              ``buffer``, ``on_push`` are also supported.

        :return: 

            * result of ``function_name`` on success
            * nill, err otherwise

    .. _router_api-replicaset_callre:

    .. method:: replicaset:callre(function_name, {argument_list}, {options})

        Call a function on the nearest available replica (distances are defined using
        ``replica.zone`` and ``cfg.weights`` matrix) with specified
        arguments,
        with preference for a replica rather than a master

>         (similar to calling vshard.router.call with prefer_replica = true).

Reviewer: ```suggestion
        (similar to calling ``vshard.router.call`` with ``prefer_replica = true``).
```

>         It is recommended to call only read-only functions using

Reviewer: ```suggestion
        It is recommended to use
```

>         ``replicaset_object:callre()``, as the function can be executed not

Reviewer: ```suggestion
        ``replicaset_object:callre()`` for calling only read-only functions, as the called function can be executed not
```
